packages/next/src/server/config.ts

```diff
@@ -292,6 +292,32 @@ function assignDefaults(
     )
   }
 
+  if (typeof result.experimental?.nextUrlServerPrefix !== 'string') {
```

Comment:
I think this is the old way of validating config. You can use zod to validate the config file here:

https://github.com/vercel/next.js/blob/c84be735f5db348931bcca71109b7f9596c28d2f/packages/next/src/server/config-schema.ts#L129